pom.xml

```diff
@@ -50,6 +50,7 @@
                 <module>apps/structure</module>
                 <module>apps/all</module>
                 <module>apps/docs</module>
+                <module>content/marketo</module>
```

Comment:
I think we want is `apps/content` and a similar structure to {ahem} structure.

Reply:
yeah, reading to what your plans are, `apps/content` seems like the way to go

Reply:
Thanks @npeltier and @auniverseaway - Will make the changes and raise a new PR. This one needs to be closed anyway, due to the Adobe org requirements.